ui/src/components/no-code/components/tasks/TaskVersion.vue

```diff
@@ -8,10 +8,16 @@
 </template>
 
 <script setup lang="ts">
-    import {computed, getCurrentInstance} from "vue";
+    import {computed} from "vue";
     import {useI18n} from "vue-i18n";
     import InputText from "../inputs/InputText.vue";
 
+    import {useMiscStore} from "override/stores/misc";
+    const miscStore = useMiscStore();
+
+    const {edition} = miscStore.configs || {};
+    const disabled = computed(() => edition === "OSS")
+
```

Comment:
More succint and explicit

```suggestion

    const disabled = computed(() => miscStore.configs?.edition === "OSS")

```